curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f0286404aefb0562a4fdf9.md

```diff
@@ -68,19 +68,19 @@ assert(code.match(/<head>[\w\W\s]*<link[\w\W\s]*\/>[\w\W\s]*<\/head>/i))
 Your `link` element should have a `rel` attribute with the value `stylesheet`.
 
 ```js
-assert.match(code, /<link[\s\S]*?rel=('|"|`)stylesheet\1/)
+assert(__helpers.removeWhiteSpace(code).match(/<.*?link.*.*?rel=('|"|`)stylesheet('|"|`).*\/>/))
 ```
 
 Your `link` element should have a `type` attribute with the value `text/css`.
 
 ```js
-assert.match(code, /<link[\s\S]*?type=('|"|`)text\/css\1/)
+assert(__helpers.removeWhiteSpace(code).match(/<.*?link.*.*?type=('|"|`)text\/css('|"|`).*\/>/))
```

Comment:
```suggestion
assert.match(code, /<link[^>]*?\s+type=('|"|`)text\/css\1/is);
```

Reply:
Hey Shaun thx for commenting! From this issue discussion i've understood were not changing the self closing request because it will cause inconsistensy in lots of other pages in this section, also i'm allmost sure that your suggested change will let invalid codes passing - i'll check that as soon as i can. Anyways ofcourse i'll wait for your response about the issue before doing anything. Thanks anyway! have a nice weekend.

Reply:
So, there is no issue with consistency, because we are not changing seed code - that is still consistent throughout. We do, however, need to allow correct code to pass.

I made some adjustments to your regex, based on the [html spec](https://html.spec.whatwg.org/#start-tags)

Specifically:
- there should **not** be any whitespace between the `<` and the tag name.
- there **should** be whitespace between attributes - which I why I removed the use of `removeWhiteSpace`

Also, a few cases of lazy matching were missing. So, I added them.

Hope this clarifies

Reply:
Hey Shaun. Sorry for late response, as iv'e had a busy weekend. 
I've seen your adjustments, and probably needed to be more specific about which codes i'm concerened about:
1. it's isnt case sensetive, so it let's wrong attributes and values passing.
2. Browsers normally allowes attributes passing even with 0 spaces between them.
3. Next steps showing the link with the self closing tags and may be a bit confusing.
I want to thank you for your time - and sorry if it feels i'm wasting your time.
My only intention is to give the best freeCodeCamp experience so other people could learn online for free as i have.

Reply:
1) As per [spec](https://html.spec.whatwg.org/#syntax-tag-name), tag names are case insensitive
2) We can only follow the spec, because browsers tend to implement things differently from one another, and if they do not follow spec, then they will likely change to do so.
3) It is not about showing or what is in the seed code. If Campers are not explicitly told to add a `/` in a void element, then we should not mark them as incorrect.

Hope this clarifies